airgun/views/settings.py

```diff
@@ -0,0 +1,31 @@
+from wait_for import wait_for
+from widgetastic.widget import Text
+
+from airgun.views.common import BaseLoggedInView, SearchableViewMixin
+from airgun.widgets import PopOverWidget, Table
+
+
+class SettingsView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Settings']")
+    table = Table(
+        './/table',
+        column_widgets={
+            'Name': Text('.//span[contains(@rel,"twipsy")]'),
```

Comment:
It is not necessary to specify locator for every column. Do it only if you like to work with widgets inside

Reply:
Hmm, that's a valid point. I will keep it in mind as this will automatically be known by the table